packages/discovery-provider/plugins/notifications/src/processNotifications/mappers/challengeReward.ts

```diff
@@ -66,6 +66,10 @@ export class ChallengeReward extends BaseNotification<ChallengeRewardRow> {
     fp: {
       title: '🎼 Create a Playlist',
       amount: 2
+    },
+    o: {
+      title: 'One shot',
+      amount: 2
```

Comment:
OOTL but this doesn't seem right?

Reply:
ah yea this amount isn't even used. it comes from the notification json